docs/migrations/web/MIGRATION-v3.md

```diff
@@ -178,6 +193,91 @@ Instead of using `.Button--square`, use a `.Button--symmetrical`.
 
 - `.Button--square` → `.Button--symmetrical`
 
+### Button: The `inverted` Modifier Removed
+
+Button `inverted` modifier was removed. Use themes to switch the button colors.
+
+### Field Group: Alignment
+
+Field Group component children are not aligned to `start` except when the `FieldGroup--fluid` modifier is used.
+
+### Header: The `inverted` Variant Removed
+
+Header `inverted` variant was removed. Instead, the `primary` variant was added to be further adjusted with themes.
+
+### Partner Logo: Stabilized
+
+Partner Logo component was stabilized.
+
+#### Migration Guide
+
+Rename all usages of `UNSTABLE_PartnerLogo` to `PartnerLogo`.
+
+- `<div class="PartnerLogo PartnerLogo--small PartnerLogo--safeArea">` → `<div class="PartnerLogo PartnerLogo--small PartnerLogo--safeArea">`
+
+### Pill: Renamed and Removed Variants
+
+Pill component variants `primary`, `secondary`, `tertiary`, `inverted` and `unselected` were removed.
+Instead, the `neutral` variant was added. The current list of variants is
+
+- `neutral`
+- `danger`
+- `informative`
+- `success`
+- `warning`
+
+#### Migration Guide
+
+Use the `neutral` or emotion variants. If you need other variants, please, let us know.
+
+### Product Logo: The `inverted` Modifier Removed
+
+Product Logo `inverted` modifier was removed. Use themes to switch the logo colors.
```

Comment:
> Use themes to switch the logo colors.

`ProductLogo` doesn't have any themeable styling. I think it's more like this:

```suggestion
The `inverted` modifier was removed from Product Logo. The component itself does not apply any color now, but you may want to swap the image source for certain themes. For example:

<div class="ProductLogo">
  <img src="path-to-logo" class="my-logo-for-light-theme" alt="Product Logo" height="60" width="120" />
  <img src="path-to-logo" class="my-logo-for-dark-theme" alt="Product Logo" height="60" width="120" />
</div>

This is a simple example, both images will be downloaded in this case. You may need to involve a more complex solution that will be performance friendly.
```

Reply:
I'm filing an issue to show how to use themeable images: https://jira.almacareer.tech/browse/DS-1519

But now I realize this is matters mainly with **dynamic themes** like dark mode… So we could leave it out here and just add a general docs?

Reply:
yes, let's do this elsewhere